app/presenters/budget_presenter.rb

```diff
@@ -18,4 +18,8 @@ def period_end_date
     return if super.blank?
     I18n.l(super)
   end
+
+  def value
+    "%.2f" % super
```

Comment:
I am interesteted to know what led you to choose this over something like `number_to_currency`? 

I think I agree that this is nice and simple and money always scares me a little bit. When I first saw this I knew what the result was but I had no idea what was going on – do you think it is too much wrap it in a method for simple folk like me, `number_to_two_decimal_places` or something - what do you think?

Reply:
I didn't know number_to_currency existed!

I can use it but that means taking it out of the presenter & using the method in the view, if you think that's OK?

Reply:
I noticed later we have a `monetary_value` that uses `to_f` I wonder if that is actually the place where we make sure it is 2 decimal places – I don't have the confidence to say right now as I haven't looked at how we handle money as yet!

Reply:
I'm a bit out of my depth as well. I know we store the value with monetary_value but the problem I'm trying to solve in the presenter is that if the value is `X` it will be presented as `X.0` but if it's `X.01` it will be presented as `X.01`.

Maybe @tahb can weigh in on the budget presenter? The issue I'm trying to solve is to stop values like `21.0` appearing on the front end.

Reply:
Thinking aloud.. given we are working with money we only care about a value to 2 decimal places: 

"0.2" is 20p - this is the case we are thinking of here
"0.23" is 23p
"0.238" the decimal type will round this to 0.24p

We are using the `decimal` data type when storing this information in the database with a scale of 2, which means when we persist the data some rounding can happen (we could address more than 2 decimals by validating the number of decimals in the value users provide on the way in and preventing them).

If this were any other type of data I'd suggest we modify the setter to store values the rest of the system can use without having to start 0 padding everywhere however when we look at the `value` field in Ruby we see the class is `BigDecimal` and the value is `0.2e0` and this is the same whether we store a trailing 0 or not. There doesn't seem to be any value add by changing the setter so I think we should solve this at the presentational layer instead. `number_to_currency` looks the perfect tool for the job:

```
number_to_currency(BigDecimal("0.2"), unit: "£") => "£0.20"
number_to_currency(BigDecimal("0.20"), unit: "£") => "£0.20"
```

I notice that transaction values don't have their values presented and could benefit from this change too!